tcb/.vscode/tasks.json

```diff
@@ -13,22 +14,24 @@
                 "showReuseMessage": true,
                 "clear": false
             },
+            "options": {
+                "cwd": "${workspaceFolder}"
+            },
             "args": [
-                "▶️ merge task finished"
+                "${workspaceFolder}/.conf/projectUpdater.ps1",
+                "${workspaceFolder}",
+                "${workspaceFolderBasename}",
+                "0"
             ],
             "dependsOrder": "sequence",
-            "dependsOn": [
-                "try-update-template"
-            ],
             "icon": {
                 "id": "repo-sync",
                 "color": "terminal.ansiCyan"
             }
         },
         {
-            "label": "try-update-template",
-            "detail": "",
-            "hide": true,
+            "label": "try-update-template-accepting-all",
+            "detail": "This task has automation steps to update the project\nbased on the template used to create it.",
```

Comment:
This task has automation steps to update the project based on the template used to create it. Be careful, this one will automatically accept all incoming changes from the updated template